extensions/resource-deployment/src/services/tools/toolBase.ts

```diff
@@ -26,6 +26,22 @@ const toolStatusLocalized: Map<ToolStatus, string> = new Map<ToolStatus, string>
 	[ToolStatus.Failed, toolStatusFailed]
 ]);
 
+export const enum dependencyType {
+	PythonAndPip3 = 'PythonAndPip3',
+	Brew = 'Brew',
```

Comment:
My preference would be to split these into separate dependencies and then or them together. I could forsee some tool needing python as a dependency but not necessarily pip

Reply:
Sure we will do that when we get to that tool. Today they are needed together in tandem. In fact in future our direction is to move away from them with dedicated installers that do not have any dependencies.